packages/angular/.eslintrc.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
          * Turn this off until we turn `strict` on in the `tsconfig.json`.
          */
         '@typescript-eslint/prefer-nullish-coalescing': 'off',
+        // making the changes for these rules leads to material changes in the package code
+        '@typescript-eslint/consistent-type-exports': 'off',
+        '@typescript-eslint/consistent-type-imports': 'off',
```

Comment:
Angular didn't handle the changes well and tbh prefer avoiding making unnecessary changes in this package